Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,45 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            for (int i = 0; i < 2; i++)
+            {
+                Console.Write("What is your name: ");
+                var name = Console.ReadLine();
+
+                Console.Write("What is your surname: ");
+                var surnname = Console.ReadLine();
+
+                Console.Write("What is your age: ");
+                var age = int.Parse(Console.ReadLine());
+
+                Console.Write("What is your weight in kilogram: ");
+                float weight = float.Parse(Console.ReadLine());
+
+                Console.Write("What is your height in metres: ");
+                float height = float.Parse(Console.ReadLine());
+
+                float bmi = GetBmi(weight, height);
+
+                Console.WriteLine($"{name}  {surnname}  is {age} years old, his weight is {weight} kg and his height is {height:F2} cm.");
+                Console.WriteLine($"His BMI is:  {bmi:N2}");
+                Console.WriteLine("");
```

Comment:
Worth having a look at escape characters, in this case \n for new line.
In this case you could trim that last line for:
`Console.WriteLine($"His BMI is:  {bmi:N2} \n");`
Just being picky, overall, good work!

Reply:
In a later commit I changed it to  `Console.WriteLine($"Now for another person {Environment.NewLine}  "); `  so it even works on a linux machine which works with `/r /n`  as far as I know

Reply:
adam-ciszewski  are you satisfied with the code now ?